res/Sectors/M1105/Folgore.txt

```diff
@@ -0,0 +1,423 @@
+0236 Spakevo              B5A5936-8 Fl Hi In (Spakevo) Da        {  2 } (889-3) [CB8D] - K  A 304 13 Hv M7 III M7 V     
+0239                      C6816S1-7 Ni Ri                        { -1 } (A50-1) [9565] - M  - 524 18 Hv A0 V            
+0240                      E220000-0 Ba De Po                     { -3 } (A00+1) [0000] - -  - 013 17 Hv M4 V K5 V       
+0332                      B423622-A Na Ni Po                     {  1 } (C55+4) [7785] - K  - 504 13 Hv M6 V            
+0333                      E332450-8 Ni Po                        { -3 } (733+3) [411B] - -  - 704 14 Hv G5 III          
+0334                      B5917S1-9 Pi He                        {  1 } (767+3) [B849] - M  - 902 13 Hv M1 V K7 V       
+0335 Tuulnurn             A6659S1-D Hi Ga Pr                     {  3 } (A89+0) [6C5D] - -  - 802 12 Hv G0 V M1 V       
+0337 Druqkask             A5839T4-C Hi Pr                        {  3 } (D88-4) [DC5A] - -  - 904  7 Hv K1 II           
+0433                      E9E9000-0 Ba                           { -3 } (900+2) [0000] - M  - 005 13 Hv M4 V            
+0436 Aladhick             B5539U5-A Hi Po                        {  3 } (888+2) [6C49] - K  - 900  8 Hv M5 V            
+0440                      E562626-5 Ni Ri                        { -2 } (350-1) [3445] - -  - 803 11 Hv G7 IV           
+0531                      C866616-7 Ga Ag Ni Ri                  {  0 } (852+1) [665C] - M  - 203 13 Hv M2 V            
+0534                      A5727T1-9 He Pi                        {  1 } (66D+1) [4859] - E  - 510  7 Hv M3 V M3 V       
+0539                      D476340-5 Lo                           { -3 } (521+2) [3155] - -  - 712 13 Hv M6 V            
+0632                      B332656-A Ni Po Na DagaW Pz            {  1 } (C53+0) [6738] - M  A 904 12 Hv M3 III          
+0637 Phleqaar             A5569T3-9 Cp Hi                        {  2 } (A89+0) [DB5A] - E  - 503  9 Hv K1 V M3 V       
+0639 Ginuran              D5829T3-7 Hi Pr                        { -1 } (783+3) [9856] - M  - 814  8 Hv M2 V            
+0640                      E210000-0 Ba                           { -3 } (700-1) [0000] - -  - 001 10 Hv F5 IV M5 V      
+0732                      C9493A6-5 Lo DagaW Pz                  { -2 } (821-1) [3165] - M  A 803 14 Hv K2 IV M6 V      
+0733                      C888714-3 Ag Ri DagaW Da               {  1 } (569-2) [A8A3] - M  A 801 10 Hv A9 V G9 V       
+0736                      C436110-8 Lo                           { -2 } (C01+3) [1179] - M  - 302 10 Hv M3 III          
+0738 Oockolkux            E6829T3-5 Hi Pr                        { -1 } (887-1) [8853] - -  - 824 11 Hv M5 III M5 V M4 V
+0740                      A540230-B He De Lo Po                  {  1 } (811-2) [135F] - -  - 500 10 Hv G9 V            
+0832 Dagaut               C784651-A Ri Ag Ni (Dagaut) Fo         { -1 } (850-2) [6552] - -  A 603 10 Hv K1 V M1 V       
```

Comment:
My linter shows this world as Ix: 2, Cx Acceptance 8, and Cx Symbols should be 5 to 15

Reply:
Fxied